### ISSUE TITLE
Remove warnings in output

### ISSUE THREAD

**Issue by MatinF** [ebroecker/canmatrix#684]
It would be great to avoid having the following 'warning output' be displayed when using canmatrix. Perhaps it can be put into a logger type output instead.

```
arxml is not supported
ldf is not supported
kcd is not supported
fibex is not supported
xls is not supported
xlsx is not supported
yaml is not supported
odx is not supported
```

**Comment by ebroecker** [comment]
should be done with https://github.com/ebroecker/canmatrix/commit/eecffcc34411c8533a00cedb154a34f03fd72082 s

does this issue still occure?

**Comment by morbult** [comment]
Yes, this is still an issue. I have suggested a fix in #748, please have a look.
